client/post-editor/post-editor.jsx

```diff
@@ -554,7 +606,9 @@ var PostEditor = React.createClass( {
 				}
 			}.bind( this );
 		}
+		this.props.autosave( callback );
```

Comment:
does it make sense to pass a callback into a reducer? I know that's what was happening with the flux action, but I suppose this is something to review when implementing the reducer?